gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/ExportPanel.java

```diff
@@ -251,7 +251,7 @@ private File buildFile(final boolean isCustomExport) {
 				if (!isCustomExport || this.includeStudentId) {
 					line.add(studentGradeInfo.getStudentEid());
 				}
-				if (!isCustomExport || this.includeStudentDisplayId) {
+				if (isCustomExport || this.includeStudentDisplayId) {
```

Comment:
The comment says you only want to export the student display ID if it is a custom export, in that case the test would need to be an 'and' not 'or'.